lib/rserve/rexp/wrapper.rb

```diff
@@ -21,9 +21,7 @@ def self.wrap(o)
             REXP::Null.new()
           when ::String
             REXP::String.new(o)
-          when Integer
-            REXP::Integer.new(o)
-          when Fixnum
+          when ::Integer
```

Comment:
Note: Just writing `when Integer` here, would only match with `REXP::Integer`. This can never occur, since `REXP::Integer` is already caught by `o.is_a? REXP` on line 13